app/src/main/java/com/github/se/wanderpals/model/data/Suggestion.kt

```diff
@@ -35,5 +36,6 @@ data class Suggestion(
     val createdAtTime: LocalTime = LocalTime.of(0, 0), // Time of creation
     val stop: Stop = Stop(), // Embed the Stop object directly
     val comments: List<Comment> = emptyList(),
-    val userLikes: List<String> = emptyList()
+    val userLikes: List<String> = emptyList(),
+    val stopStatus: CalendarUiState.StopStatus = CalendarUiState.StopStatus.NONE // Default value
```

Comment:
Not required, this variable is only accessed in FirebaseSuggestion to store and retrieve it, never accessed again.
It also doesn't make sense to have the suggestion hold info about the stop after the suggestion has been accepted.
Needs to be removed.